ui/components/app/confirm-subtitle/confirm-subtitle.js

```diff
@@ -1,27 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ERC1155, ERC721 } from '@metamask/controller-utils';
 import { useSelector } from 'react-redux';
 
 import { SECONDARY } from '../../../helpers/constants/common';
-import { Text } from '../../component-library';
 import {
   Color,
   FONT_WEIGHT,
   TextVariant,
 } from '../../../helpers/constants/design-system';
-import UserPreferencedCurrencyDisplay from '../user-preferenced-currency-display';
 import { getShouldShowFiat } from '../../../selectors';
 import { useTransactionInfo } from '../../../hooks/useTransactionInfo';
+import { Text } from '../../component-library';
+import UserPreferencedCurrencyDisplay from '../user-preferenced-currency-display';
 
 const ConfirmSubTitle = ({
   txData,
   hexTransactionAmount,
   subtitleComponent,
+  assetStandard,
 }) => {
   const shouldShowFiat = useSelector(getShouldShowFiat);
   const { isNftTransfer } = useTransactionInfo(txData);
 
-  if (!shouldShowFiat && !isNftTransfer) {
+  if (
+    !shouldShowFiat &&
+    !isNftTransfer &&
+    assetStandard !== ERC1155 &&
```

Comment:
Should we make the `assetStandard !== ERC1155 && && assetStandard !== ERC721` a helper function?  I think we use this multiple times in the codebase.  Maybe a helper function called `isNftStandard`?

Reply:
Done